environments.json

```diff
@@ -2633,15 +2640,15 @@
     "short": "Edge 87",
     "release": "2020-11-19",
     "equals": "chrome87",
-    "obsolete": true
+    "obsolete": false
```

Comment:
why?

Reply:
I forgot about it, but I do not understand why, since I added a few omitted versions to each browser, I should not add "obsolete" to "true" or "very" to the older ones. After all, without this, the readability of the page will be terribly impaired.

Reply:
right but this specific change is making an obsolete version NOT be obsolete